viadot/flows/sap_rfc_to_adls.py

```diff
@@ -10,13 +10,15 @@ class SAPRFCToADLS(Flow):
     def __init__(
         self,
         name: str,
-        query: str = None,
+        query: str,
         rfc_sep: str = None,
         rfc_replacement: str = "-",
         func: str = "RFC_READ_TABLE",
         rfc_total_col_width_character_limit: int = 400,
         rfc_unique_id: List[str] = None,
         sap_credentials: dict = None,
+        saprfc_credentials_key: str = "SAP",
+        env: str = "PROD",
```

Comment:
```suggestion
        env: str = "DEV",
```
Change to 'DEV' everywhere.